section4/exercises/burrito.rb

```diff
@@ -11,9 +11,28 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    @toppings += toppings
+  end
+
+  def remove_topping(toppings)
+    @toppings -= toppings
```

Comment:
`toppings` is array data, so those operators won't work with that data type. Do some research into array prototype methods in ruby to find some ways to work with arrays,